src/containers/Tenant/Query/QueryEditor/keybindings.ts

```diff
@@ -0,0 +1,17 @@
+import type Monaco from 'monaco-editor';
+
+export function getKeyBindings(monaco: typeof Monaco) {
+    const {KeyMod, KeyCode} = monaco;
+
+    const ctrlKey = KeyMod.CtrlCmd;
+
+    /* eslint-disable no-bitwise */
+    return {
+        sendQuery: ctrlKey | KeyCode.Enter,
+        sendSelectedQuery: ctrlKey | KeyMod.Shift | monaco.KeyCode.Enter,
+        selectPreviousQuery: ctrlKey | KeyCode.UpArrow,
+        selectNextQuery: ctrlKey | KeyCode.DownArrow,
+        saveQuery: ctrlKey | KeyCode.KeyS,
```

Comment:
It conflicts with default system or browser save key. It saves current page instead of query

Reply:
If editor is focused, it will save current query. If not, page will be saved. Please try it!

Reply:
I tried. For me it saves the page, not the query

Reply:
It works. Maybe I did something wrong